### ISSUE TITLE
Organise hardware to run livestream

### ISSUE THREAD

**Issue by RealOrangeOne** [srobo/tasks#1049]
This ticket relates to providing the actual machine that will be used for the livestream.
Requirements: - Video mixer PC (With dedicated GPU, capable of running OBS) - 2× HDMI capture card - 2× preview monitors - 2× cameras (depending on feed plan) - Audio mixers

### Original

[comp/livestream/hardware](https://github.com/srobo/recurring-tasks/blob/master/comp/livestream/hardware.yaml)

### Dependencies

 * #1048 Plan video feeds

**Comment by antonnikitin97** [comment]
I can provide my PC, it has a RTX 3070, 32GB RAM. I can also provide monitors.

**Comment by RealOrangeOne** [comment]
I suggest we refine the list of hardware we need at/after the virtual competition livestream.

**Comment by antonnikitin97** [comment]
Current list of hardware I can provide:
1x Desktop PC + mouse/keyboard + usb hub
3x monitors (2x 24 inch, 1x 27 inch)
1x blue yeti mic
1x webcam

**Comment by RealOrangeOne** [comment]
I can bring a spare mic and arm as a secondary input if needed. I also have another Yeti somewhere. There's also a webcam I can bring as a second camera feed should we want it.

@WillB97 I believe has a camera (and associated cabling / capture) for the skycam. I believe you also know people who would be interesting to talk to about livestream hardware?

We'll also need an audio interface of some kind of the arena audio feed (commentators, mixtape etc). Perhaps we can reuse @prophile 's that we're using for the virtual competition.

@sedders123 I believe has a streamdeck which has been useful in the past.

Headphones are also useful, partly for monitoring audio, partly for passive sound isolation from the rest of the venue to hear yourself talk.

**Comment by antonnikitin97** [comment]
@WillB97 has put me in contact with Jamie to discuss multi-cam setups for the physical competition.

**Comment by antonnikitin97** [comment]
Copying info from #1048.
TL;DR: Equipment is available for us to use (cameras). Need to find out how many people have capture cards, plan what scenes we want, if we want NDI: comp network access.

**Comment by antonnikitin97** [comment]
Following on from #1048 I spoke to Jamie yesterday and he is happy to supply cameras and associated cables/capture cards.

I will update this ticket when I have a definitive list of equipment.

**Comment by antonnikitin97** [comment]
I'm going to close this ticket since all audio/visual equipment has been organised.

@Tyler-Ward has said we can borrow an audio mixer from Denis if needed.

**Comment by WillB97** [comment]
Could you update the freight list https://docs.google.com/spreadsheets/d/13Eao155b_ITvRhGjU_PrIv_EYo8ynfJTbagcs_rIhY8/edit?usp=sharing
